Dockerfile

```diff
@@ -28,23 +28,27 @@ RUN npm install -g bower less
 
 # install IPython 2.x branch
 WORKDIR /srv
-RUN git clone --depth 1 -b 2.x https://github.com/ipython/ipython.git
+RUN git clone --depth 1 -b 3.x https://github.com/ipython/ipython.git
 WORKDIR /srv/ipython
 RUN git submodule init && git submodule update
-RUN pip install .
+RUN pip install -e .[notebook]
 
 RUN pip install invoke
 
-ADD . /srv/nbviewer/
+ADD ./requirements.txt /srv/nbviewer/
 WORKDIR /srv/nbviewer
 
-RUN invoke bower
-RUN invoke less
-
 RUN pip install -r requirements.txt
 
+ADD ./tasks.py /srv/nbviewer/
+ADD ["/nbviewer/static/bower.json", "/nbviewer/static/.bowerrc", "/srv/nbviewer/nbviewer/static/"]
```

Comment:
Ah, it's intended to be `["<src>"... "<dest>"]`. Should these be `./nbviewer/static/...`?